core/tests/webdriverio_utils/general.js

```diff
@@ -305,6 +305,17 @@ var goOffline = async function() {
   });
 };
 
+// Call func. If func errors, print the full stack trace,
+// including the lines from the caller function.
+// This can help us chain stack traces across async promises.
+// @param {function} func Function you want to call.
+// @param {string} errStack Put this: new Error().stack.
```

Comment:
Can you explain the effects of this -- i.e. why it needs to be done?

Reply:
done.